util/argo/argo.go

```diff
@@ -421,6 +435,10 @@ func GetAppProjectByName(name string, projLister applicationsv1.AppProjectLister
 	for _, repo := range repos {
 		project.Spec.SourceRepos = append(project.Spec.SourceRepos, repo.Repo)
 	}
+	clusters := retrieveScopedClusters(name, db, ctx)
+	for _, cluster := range clusters {
+		project.Spec.Destinations = append(project.Spec.Destinations, argoappv1.ApplicationDestination{Server: cluster.Server, Namespace: "*"})
```

Comment:
Cluster might be connected in a "namespaced" mode - `cluster.Namespaces` is not empty.  In this case only these namespaces should be auto-added to the destination. E.g.

```golang
if len(cluster.Namespaces) == 0 {
  project.Spec.Destinations = append(project.Spec.Destinations, argoappv1.ApplicationDestination{Server: cluster.Server, Namespace: "*"})
} else {
  for _, ns := range cluster.Namespaces { 
    project.Spec.Destinations = append(project.Spec.Destinations, argoappv1.ApplicationDestination{Server: cluster.Server, Namespace: ns})
  }
}
```